apps/smart-contracts/token/test/BlocklistTransferHook.test.ts

```diff
@@ -143,4 +143,57 @@ describe('BlocklistTransferHook', () => {
       expect(await blocklistTransferHook.getBlocklist()).to.eq(JUNK_ADDRESS)
     })
   })
+
+  describe('hook', () => {
+    let source: SignerWithAddress
+    let destination: SignerWithAddress
```

Comment:
```suggestion
    let sender: SignerWithAddress
    let recipient: SignerWithAddress
```
Per the discussion on discord and our call today.